packages/config/config/devices/0x027a/zse11.json

```
@@ -0,0 +1,111 @@
+// Zooz ZSE11
+// Q Sensor
+{
+	"manufacturer": "Zooz",
+	"manufacturerId": "0x027a",
+	"label": "ZSE11",
+	"description": "Q Sensor",
+	"devices": [
+		{
+			"productType": "0x0201",
+			"productId": "0x0006"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"metadata": {
+		"inclusion": "Put your Z-Wave hub into inclusion mode and click the Z-Wave button 3 times as quickly as possible. The LED indicator will start blinking to confirm inclusion mode. The sensor will automatically pair as a repeater if connected to USB power, no special button sequence required.",
+		"exclusion": "Bring the sensor within direct range of your Z-Wave gateway (hub). Put the Z-Wave hub into exclusion mode. Press and release the Z-Wave button 3 times quickly.",
+		"wakeup": "Press and hold the Z-Wave Button for 3 seconds to wake the sensor up manually"
+	},
+	"paramInformation": {
+		"12": {
+			"label": "Motion Sensor Sensitivity",
+			"description": "8 is the highest sensitivity",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 8,
+			"defaultValue": 6,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"13": {
+			"label": "Motion Detection Timeout",
+			"valueSize": 2,
+			"unit": "seconds",
+			"minValue": 10,
+			"maxValue": 3600,
+			"defaultValue": 30,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"19": {
+			"label": "LED Indicator for Motion",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Off",
+					"value": 0
+				},
+				{
+					"label": "Flash LED on Motion",
+					"value": 1
+				}
+			]
+		},
```

Just a small nit:
```suggestion
		"19": {
			"label": "Flash LED on Motion",
			"valueSize": 1,
			"minValue": 0,
			"maxValue": 1,
			"defaultValue": 1,
			"readOnly": false,
			"writeOnly": false,
			"allowManualEntry": false,
			"options": [
				{
					"label": "Disable",
					"value": 0
				},
				{
					"label": "Enable",
					"value": 1
				}
			]
		},
```
allows us to make these on/off parameters booleans (toggles) in future versions.

---

Good eye, and thank you for using the suggestion feature. I haven't seen these tools before, but you make PRs very easy to manage with the bots + suggestions!